numpy/random/mtrand/distributions.c

```diff
@@ -315,7 +316,7 @@ long rk_binomial_btpe(rk_state *state, long n, double p)
     v = v*(u-p3)*lamr;
 
   Step50:
-    k = fabs(y - m);
+    k = labs(y - m);
```

Comment:
There is the corner case where `labs(y - m)` is negative, so maybe the safer thing here is to cast `y - m` to float.